crates/swc_ecma_preset_env/tests/fixtures/corejs3/usage-promise-finally/output.mjs

```diff
@@ -2,6 +2,6 @@ import "core-js/modules/es.object.to-string";
 import "core-js/modules/es.promise";
 import "core-js/modules/es.promise.finally";
 var p = Promise.resolve(0);
-p.finally(function() {
+p["finally"](function() {
```

Comment:
I think it's also happening for corejs3

Reply:
Ah, my bad.
I misread corejs-3 and corejs-2

Reply:
I will look at this in near future, we need investigate it, also we will need this logic for css minificator, because it can be done based on browserslist, so maybe we need to move it to separate crate, but let's investigate current problem ehre